source/_integrations/sensorpush_cloud.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: SensorPush Cloud
+description: Instructions on how to integrate SensorPush Cloud devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2024.8
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@sstallion'
+ha_domain: sensorpush_cloud
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [SensorPush Cloud](https://www.sensorpush.com/) devices into Home Assistant.
+
+## Activation is required
+
+Sensor entities (temperature, humidity, barometric pressure) will not be available to Home Assistant until you have activated the device with the SensorPush app on iOS or Android.
+
+To activate API access, log in to the [Gateway Cloud Dashboard](https://dashboard.sensorpush.com/) and agree to the terms of service.
+
+## Supported devices
+
+- [G1 WiFi Gateway](https://www.sensorpush.com/products/p/g1-gateway)
+- [HT1 Temperature and Humidity Smart Sensor](https://www.sensorpush.com/products/p/ht1)
+- [HT.w Water-Resistant Temperature / Humidity Smart Sensor](https://www.sensorpush.com/products/p/ht-w)
+- [HTP.xw Extreme Accuracy Water-Resistant Temperature / Humidity / Barometric Pressure Smart Sensor](https://www.sensorpush.com/products/p/htp-xw)
+
+{% include integrations/config_flow.md %}
```

Comment:
below here, could you add a section ##Sensors, listing the names of all the sensors, together with a short description?